front/components/spaces/EditSpaceManagedDatasourcesViews.tsx

```diff
@@ -276,12 +276,12 @@ export function EditSpaceManagedDataSourcesViews({
         systemSpaceDataSourceViews={filterSystemSpaceDataSourceViews}
         onSave={async (selectionConfigurations) => {
           if (
-            !(await confirmPrivateNodesSync({
+            await confirmPrivateNodesSync({
```

Comment:
This is a bit misleading but this method returns true if there is no private nodes, or if the user validates.